firebase-vscode/src/data-connect/emulator.ts

```diff
@@ -48,9 +50,26 @@ export class DataConnectEmulatorController implements vscode.Disposable {
 
   private async connectToPostgres() {
     const rc = firebaseRC.value?.tryReadValue;
+    let localConnectionString =
+      rc?.getDataconnect()?.postgres?.localConnectionString;
+    if (!localConnectionString) {
+      const dataConnectConfigsValue =
+        await firstWhereDefined(dataConnectConfigs);
+      let dbname = "postgres";
+      const postgresql =
+        dataConnectConfigsValue?.tryReadValue.values[0]?.value?.schema
```

Comment:
we can make this available in the ResolveDataConnectConfig class.

Reply:
I'll clean this up after the release :)

Reply:
i can do it :) much clean up to be done

Reply:
Sure, I'll leave it up to you then